cypress_test/integration_tests/desktop/writer/top_toolbar_spec.js

```diff
@@ -199,6 +199,45 @@ describe('Top toolbar tests.', function() {
 		cy.get('#copy-paste-container ul')
 			.should('exist');
 	});
+
+	it('Increase Indent.', function() {
+		cy.get('#toolbar-up .w2ui-scroll-right')
+			.click();
+		helper.reselectAllText();
+
+		cy.get('#copy-paste-container p')
+			.should('have.attr', 'style', 'margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%');
+	
+		cy.get('#tb_editbar_item_incrementindent')
+			.click();
+
+		helper.reselectAllText();
+
+		cy.get('#copy-paste-container p')
+			.should('have.attr', 'style', 'margin-left: 0.49in; margin-bottom: 0in; font-style: normal; font-weight: normal; line-height: 100%');
+	});
+
+	it('Decrease Indent.', function() {
```

Comment:
You can rename this test case like 'Increase/decrease indent.' and remove the other test above. It's better to avoid code duplication here. It's OK to test two things in one test case.